joplin/base/wagtail_hooks.py

```diff
@@ -61,7 +61,8 @@ def configure_main_menu(request, menu_items):
                      [
                          'explorer',
                          'settings',
-                         'snippets'
+                         'snippets',
+                         'reports'
```

Comment:
This is a new thing that wagtail has, which we don't do anything with, so I'm assuming we'll want to keep hiding it.

Reply:
Curious what the reports.. report. could be really interesting.

Reply:
Check it out in https://odd-tail-bakery.herokuapp.com/admin/

admin : admin
PW: changeme